shared/dataflow/codeql/dataflow/DataFlow.qll

```diff
@@ -169,8 +199,17 @@ signature module InputSig {
     string toString();
   }
 
+  /**
+   * Holds if the parameter position `ppos` matches the argument position
+   * `apos`.
+   */
   predicate parameterMatch(ParameterPosition ppos, ArgumentPosition apos);
 
+  /**
+   * Holds if there is a simple local flow step from `node1` to `node2`. This
+   * is the local flow predicate that's used as a building block in global
+   * data flow. It may have less flow than the `localFlowStep` predicate.
```

Comment:
The `localFlowStep` predicate that most (all?) languages expose isn't in scope here as it's not relevant to the data flow library, so we shouldn't mention it here. Also, saying that it's a building block of global data flow is somewhat redundant, as that's pretty much the case for all of these predicates in this signature. Instead we can add some details about these being value-preserving and elaborate a bit what local actually means.
```suggestion
   * Holds if there is a simple local flow step from `node1` to `node2`. These
   * are the value-preserving intra-callable flow steps.
```

Reply:
Thanks!